src/theme/components/multiSelect.ts

```diff
@@ -0,0 +1,97 @@
+const multiSelectWidths = {
+  default: {
+    width: {
+      base: "100%",
+      md: "250px",
+    },
+  },
+  fitContent: {
+    width: {
+      base: "100%",
+      md: "fit-content",
+    },
+    minWidth: {
+      md: "175px",
+    },
+  },
+  full: {
+    width: "100%",
+  },
+};
+
+const MultiSelect = {
+  parts: [
+    "menuButton",
+    "menuContainer",
+    "menu",
+    "menuChildren",
+    "selectedItemsCountButton",
+  ],
+  baseStyle: ({ width = "default", isBlockElement }) => ({
+    ...multiSelectWidths[width],
+    position: "relative",
+    margin: {
+      base: ".75rem 0",
+      md: "0 2em 0 0",
+    },
+    menuButton: {
```

Comment:
I think you should create a new file for the `MultiSelectMenuButton` component's style. That file can be `multiSelectMenuButton.ts` and should include the styles for `menuButton` and `selectedItemsCountButton`. Then, in `MultiSelectMenuButton.tsx`, you can import _just_ the styles you need for the button `const styles = useMultiStyleConfig("MultiSelectMenuButton", {});`.

The button styles are also not necessary in the "dialog" or "listbox" variants so it'll help separate the styles.

Reply:
The style file fro menu button has been added.